### ISSUE TITLE
val_loss does not decrease

### ISSUE THREAD

**Issue by robin0506** [emilwallner/Screenshot-to-code-in-Keras#33]
I trained model the bootstrap version for about 50 epochs, which contains 10 pictures.
Specifically, I use 90 percent sample as train sample and 10 percent sample as test sample.
Till now, the loss function decrease from around 2.7 to around 0.05 whereas the val_loss keeps almost the same, ranging from 1 to 3.
I am quite confused. 
should I use more samples? (now I only use 10 pictures in the code as samples)
or should I make more training?
I appreciate your help.

**Comment by emilwallner** [comment]
Well done! You need more samples. You can use the dataset I link to in the readme. The dataset includes 1500 test screenshots and 250 validation images. For each screenshot there are on average 65 tokens, resulting in 96925 training examples.

**Comment by robin0506** [comment]
Cool, thank you for your guiding

**Comment by laptopmutia** [comment]
@emilwallner

 hi emill how you get the number? 96926 examples?

my understanding is

if you have 1500 test screenshots + complete tokens then it means 1500 examples?

**Comment by emilwallner** [comment]
It predicts token by token, so if it has 65 tokens, it needs to run 65 predictions for one screenshot. I took the average amount of tokens times the number of screenshots. (~65 * 1500)
